sos/report/plugins/openstack_nova.py

```diff
@@ -118,6 +119,11 @@ def setup(self):
             self.var_puppet_gen + "_placement/etc/httpd/conf.d/",
             self.var_puppet_gen + "_placement/etc/httpd/conf.modules.d/*.conf",
             self.var_puppet_gen + "_placement/etc/my.cnf.d/tripleo.cnf",
+            self.var_puppet_gen + "_metadata/etc/nova/",
+            self.var_puppet_gen + "_metadata/etc/httpd/conf/",
+            self.var_puppet_gen + "_metadata/etc/httpd/conf.d/",
+            self.var_puppet_gen + "_metadata/etc/httpd/conf.modules.d/*.conf",
+            self.var_puppet_gen + "_metadata/etc/my.cnf.d/tripleo.cnf",
```

Comment:
I am fine with this current approach, but just in case there would be yet another subpath (like `/` or `_placement/` or `_metadata/`), from which we collect the same set of specs: wouldnt it be easier to have something like:

    paths=['', '_placement', '_metadata']
    specs = ['/etc/nova/', '/etc/my.cnf.d/tripleo.cnf', .......]
    self.add_copy_spec(['%s%s%s' % (self.var_puppet_gen, p, s) for p in paths for s in specs])

(this won't add plain `/etc/nova` from line 109/110)

Re-reading the other specs here: should be `_libvirt` dealt likewise `_placement` and/or `_metadata`?

Reply:
libvirt does not require extra work for httpd-like paths